setup.py

```diff
@@ -0,0 +1,31 @@
+"""
+Utilities for testing Digital Marketplace apps.
+"""
+import re
+import ast
+from setuptools import setup, find_packages
+
+
+_version_re = re.compile(r'__version__\s+=\s+(.*)')
+
+with open('dmtestutils/__init__.py', 'rb') as f:
+    version = str(ast.literal_eval(_version_re.search(
+        f.read().decode('utf-8')).group(1)))
+
+setup(
+    name='digitalmarketplace-test-utils',
+    version=version,
+    url='https://github.com/alphagov/digitalmarketplace-test-utils',
+    license='MIT',
+    author='GDS Developers',
+    description=__doc__.strip().split('\n')[0],
+    long_description=__doc__,
+    packages=find_packages(),
+    include_package_data=True,
+    install_requires=[
```

Comment:
Do we need to put in `Flask` as a dependency here? Or `Flask-Login`?

Reply:
So my thought process was: if you are testing something that requires X (e.g. where X==Flask) then you will already have that as a dependency. If you're not testing something that requires X, and don't already have it as a dependency, then bringing it in wouldn't be at all helpful or necessary.

So probably the deps on this module can be minimal, but maybe I should comment that here?

Reply:
If this is a principle we're including from the outset, it might be better put in the README rather than a comment above a particular piece of code, so that's it's more obviously meant to be applicable to the entire repo.